libpromises/failsafe.cf

```diff
@@ -114,6 +118,12 @@ bundle agent failsafe_cfe_internal_update
         depth_search => failsafe_recurse("inf"),
         classes => failsafe_results("namespace", "inputdir_update");
 
+      "$(sys.workdir)/modules"
```

Comment:
Oh right, good catch! On the hub the source and destination are the same, so we should not attempt a copy there.